docs/reference/0-roadmap/api-faq.mdx

```diff
@@ -152,14 +139,21 @@ Management Service.
 
 Yes &mdash; if referring to the Admin API
 [specification](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-API-Standards/blob/main/api-specifications/admin-api/admin-api-2.2.0.yaml).
-No &mdash; if referring to the specific software application called "Admin API". The
-Data Management Service will have a different database system than the ODS/API,
-including restructuring of the data currently housed in the `EdFi_Admin` and
-`EdFi_Security` databases. The project will introduce a new application,
-tentatively called the Ed-Fi Configuration Service. The plan is to implement the
-Admin API specification (version 2), so that applications and scripts developed
-on this specification can interact seamlessly either with the platform of today
-or of tomorrow.
+No &mdash; if referring to the specific software application called "Admin API".
+The Data Management Service will have a different database system than the
+ODS/API, including restructuring of the data currently housed in the
+`EdFi_Admin` and `EdFi_Security` databases. The project will introduce a new
+application, tentatively called the Ed-Fi Configuration Service. The plan is to
+implement the Admin API specification (version 2), so that applications and
+scripts developed on this specification can interact seamlessly either with the
+platform of today or of tomorrow.
+
+## Q: How can I / my team prepare for the upgrade
+
+Start by identifying all direct ODS integrations in your environment (e.g. ETLs
```

Comment:
Since many people refer to the API application as "ODS", suggest "direct ODS database integrations".